network-api/networkapi/buyersguide/models.py

```diff
@@ -367,74 +399,105 @@ class Product(models.Model):
+                FieldPanel('privacy_policy_reading_level'),
+                FieldPanel('privacy_policy_reading_level_url'),
+            ],
+            heading="Minimum Security Standards",
+            classname="collapsible"
+        ),
+        MultiFieldPanel(
+            [
+                FieldPanel('camera_device'),
+                FieldPanel('camera_app'),
+                FieldPanel('microphone_device'),
+                FieldPanel('microphone_app'),
+                FieldPanel('location_device'),
+                FieldPanel('location_app'),
+            ],
+            heading="It uses your...",
+            classname="collapsible"
         ),
-        MultiFieldPanel([
-            FieldPanel('camera_device'),
-            FieldPanel('camera_app'),
-            FieldPanel('microphone_device'),
-            FieldPanel('microphone_app'),
-            FieldPanel('location_device'),
-            FieldPanel('location_app'),
-        ],
-            heading="Can it spy on me?",
+        MultiFieldPanel(
+            [
+                FieldPanel('how_does_it_share'),
+                FieldPanel('delete_data'),
+                FieldPanel('delete_data_helptext'),
+                FieldPanel('child_rules'),
+                FieldPanel('child_rules_helptext'),
+                FieldPanel('collects_biometrics'),
+                FieldPanel('collects_biometrics_helptext'),
+                FieldPanel('user_friendly_privacy_policy'),
+            ],
+            heading="How does it handle privacy",
             classname="collapsible"
         ),
-        MultiFieldPanel([
-            FieldPanel('uses_encryption'),
-            FieldPanel('uses_encryption_helptext'),
-            FieldPanel('privacy_policy_url'),
-            FieldPanel('privacy_policy_reading_level'),
-            FieldPanel('privacy_policy_reading_level_url'),
-            FieldPanel('privacy_policy_helptext'),
-            FieldPanel('share_data'),
-            FieldPanel('share_data_helptext'),
-        ],
-            heading="What does it know about me?",
+        MultiFieldPanel(
+            [
+                FieldPanel('worst_case'),
+            ],
+            heading="The worst case is...",
             classname="collapsible"
         ),
-        MultiFieldPanel([
-            FieldPanel('must_change_default_password'),
-            FieldPanel('must_change_default_password_helptext'),
-            FieldPanel('security_updates'),
-            FieldPanel('security_updates_helptext'),
-            FieldPanel('delete_data'),
-            FieldPanel('delete_data_helptext'),
-            FieldPanel('child_rules'),
-            FieldPanel('child_rules_helptext'),
-        ],
-            heading="Can I control it?",
+        MultiFieldPanel(
+            [
+                InlinePanel(
+                    'privacy_policy_links',
+                    label='link',
+                    min_num=1,
+                    max_num=3,
```

Comment:
this is why `Product` had to become a clustered model instead of a plain model: this inline panel now lets staff pick 1-to-3 privacy policy links per product, rather than only showing a single one.